lib/testutils/e2e_describe.go

```diff
@@ -45,7 +45,7 @@ func E2eDatastoreDescribe(description string, datastores DatastoreType, body fun
 			func() {
```

Comment:
I think this whole etcdv2 branch and the DatastoreEtcdV2 option needs to go too.